traffic_ops/traffic_ops_golang/types/types_test.go

```diff
@@ -156,6 +156,18 @@ func TestUpdateInvalidType(t *testing.T) {
 	}
 }
 
+func TestDeleteInvalidType(t *testing.T) {
+	invalidDeleteType := createDummyType("other")
+
+	err, _, statusCode := invalidDeleteType.Delete()
+	if err == nil {
+		t.Fatalf("expected delete type to have an error")
+	}
+	if statusCode != http.StatusNotFound {
```

Comment:
Should have caught this before approving—This still needs to be `http.StatusBadRequest` instead of `http.StatusNotFound`. Test fails

Reply:
https://github.com/apache/trafficcontrol/pull/4610